src/components/landing/signIn/SignIn.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Input from "@common/input/Input";
+import Button from "@common/button/Button";
+import styles from "./SignIn.module.css";
+
+const SignUp = () => {
```

Comment:
Signin or Signup? I think there is a typo considering the name of the file

Reply:
True! I'd adjusted it